builder/dockerfile/copy.go

```diff
@@ -56,6 +56,7 @@ type copyInstruction struct {
 	cmdName                 string
 	infos                   []copyInfo
 	dest                    string
+	flChown                 string
```

Comment:
`s/flChown/idPair/` or something like that. At this point I don't think it's relevant that this came from a flag.